### DISCUSSION TITLE
Welcome to restoration_framework Discussions!

### DISCUSSION THREAD

**Discussion by NewGraphEnvironment** [NewGraphEnvironment/restoration_framework#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  ➡️ We can **convert issues to discussions** and we can propose new issues through discussions 

Please note that this project is released with a [Contributor Code of Conduct](https://contributor-covenant.org/version/2/1/CODE_OF_CONDUCT.html). By contributing to this project, you agree to abide by its terms.

**Comment by NewmanNJN** [comment]
#14 Hey everyone, I'm attempting to add a couple potential restoration sites today before the meeting. Has anyone had luck doing it throug GitHub as of yet?
